libs/platform/src/lib/components/action-button-group/action-button-group.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'fdp-action-button-group',
+    templateUrl: './action-button-group.component.html',
+    styleUrls: ['./action-button-group.component.scss']
+})
+export class ActionButtonGroupComponent {
+
+}

```

Comment:
IS it not better to make it as a directive? It has only 1 class added to 1 element and `ng-content`

Reply:
In future we're planing to implement the , overflow of actions that will be used in shell bar, action bar and panel.